src/main/reaktivity/protocol.idl

```diff
@@ -111,7 +122,7 @@
         struct MqttPingResp
         {
             uint8 packetType = 0xd0;
```

Comment:
Consider moving packet type fixed value to `enum` instead, requires https://github.com/reaktivity/nukleus-maven-plugin/pull/89.

```
enum MqttPacketType (uint8)
{
    CONNECT (1),
    CONNACK (2),
    PUBLISH (3),
    PUBACK (4),
    PUBREC (5),
    PUBREL (6),
    PUBCOMP (7),
    SUBSCRIBE (8),
    SUBACK (9),
    UNSUBSCRIBE (10),
    UNSUBACK (11),
    PINGREQ (12),
    PINGRESP (13),
    DISCONNECT (14),
    AUTH (15)
}
```